CollAction/Controllers/ProjectsController.cs

```diff
@@ -60,14 +61,21 @@ public ViewResult StartInfo()
             return View(model);
         }
 
-        public async Task<IActionResult> Details(int? id)
+        // GET: Projects/Details/5
+        public async Task<IActionResult> DetailsById(int? id)
```

Comment:
Any particular reason why it's a nullable int?

Reply:
Not really, only that GetProjectById takes nullable int so it's type compatible already.

Reply:
Null is probably not a sensible parameter, since it'll only yield a 404. It being nullable is probably a small pre-existing bug. Good idea to fix as well.